primitives/state-machine/src/overlayed_changes/changeset.rs

```diff
@@ -405,16 +405,19 @@ impl OverlayedChangeSet {
 	}
 
 	/// Set all values to deleted which are matched by the predicate.
-	///
+	/// The function returns the number of keys cleared
```

Comment:
sidenote it is better practice to have a short description of the method then an empty line then some more description,
Rustdoc use the first short description in some context and show full doc in some other.

Reply:
Ok...